modules/doc/content/application_development/profiling.md

```diff
@@ -7,6 +7,33 @@ installation process.  Although it works best on Linux platforms, it also
 works reasonably well on Mac OS.  Instruments also works well for profiling
 applications on Mac systems.
 
+## Install Google's gperftools
+
+If your environment package from the MOOSE installation process has included Google's gperftools,
+you can skip this step.
+To install Google's gperftools on your own, our recommended procedure is
+
```

Comment:
We should verify if libunwind is optional or not. I always use libunwind.
 
 I always install https://github.com/google/pprof 
 
 Not sure the old pprof inside of gperftools is good enough or not

Reply:
> Not sure the old pprof inside of gperftools is good enough or not

It is not maintained any more. All development focuses on https://github.com/google/pprof

Reply:
I did not install `pprof` and `libunwind` before. Care to give the detailed commands for building them, so that I can add them here? Or you can push a commit on top of mine.

Reply:
Yes I can push a commit regarding these things. I will also push about `binutils`